fm4tlp/test.py

```diff
@@ -60,6 +61,11 @@
 from utils import train_test_helper
 from utils import utils
 
+if 'gfile' not in sys.modules:
```

Comment:
You do not need to copy this pattern from the other files. Please remove it and re-insert the `tf.io.*` calls throughout.